File: php-pr/51/php/code-quality-error-handling/PropertyAccessor.php
<?php

class PropertyAccessor
{
    public function getProperty($object, $property)
    {
        // {fact rule=code-quality-error-handling@v1.0 defects=1}
        try {
            return $object->$property;
        } catch (Exception $e) {
            // Silent failure
        }
        // {/fact}
    }
}

File: kotlin-pr/50/kotlin/code-quality-error-handling/PropertyAccessor.kt
import kotlin.reflect.full.memberProperties

class PropertyAccessor {
    fun getProperty(obj: Any, propertyName: String): Any? {
        // {fact rule=code-quality-error-handling@v1.0 defects=1}
        return try {
            val property = obj::class.memberProperties.find { it.name == propertyName }
            property?.getter?.call(obj)
        } catch (e: Exception) {
            null // Silent failure
        }
        // {/fact}
    }
}